tests/test_package.py

```diff
@@ -114,16 +119,28 @@ def test_package_upload(self, mock_conn, package_file: str, runner) -> None:
         )
 
     @pytest.fixture
-    def packages_directory(self):
-        path = os.path.join(os.getcwd(), ".packages")
-        os.mkdir(path)
-        yield path
+    def temp_dir(self):
+        initial_dir = os.getcwd()
+        tmp = tempfile.TemporaryDirectory()
+        os.chdir(tmp.name)
+        yield tmp
+        os.chdir(initial_dir)
+        tmp.cleanup()
 
     @pytest.fixture
     def package_file(self):
         with tempfile.TemporaryDirectory() as tmp:
             yield create_named_file("app.zip", tmp, [])
 
+    @pytest.fixture
+    def dot_packages_directory(self, temp_dir):
```

Comment:
1. You can create directory with subdirectories https://stackoverflow.com/questions/273192/how-do-i-create-a-directory-and-any-missing-parent-directories
2. You don't have to chdir there
3. You use it in only one test, it has to be fixture?

Reply:
1. Done
2. Done
3. I`m thinking hard about adding another test, that would use it :)